netmaster/master/endpointGroup.go

```diff
@@ -64,6 +67,32 @@ func CreateEndpointGroup(tenantName, networkName, groupName string) error {
 		return err
 	}
 
+	// check epg range is with in network
+	if len(ipPool) > 0 {
+		if netutils.IsIPv6(ipPool) == true {
+			return fmt.Errorf("ipv6 address range is not supported")
+		}
+
+		if err = netutils.ValidateNetworkRangeParams(ipPool, nwCfg.SubnetLen); err != nil {
+			return fmt.Errorf("invalid ip-pool %s", ipPool)
+		}
+
+		addrRangeList := strings.Split(ipPool, "-")
+		if _, err := netutils.GetIPNumber(nwCfg.SubnetIP, nwCfg.SubnetLen, 32, addrRangeList[0]); err != nil {
+			return fmt.Errorf("ip-pool %s is not part of network %s/%d", ipPool, nwCfg.SubnetIP,
```

Comment:
Please change the error message to "Bad ip-pool %s. Epg ip-pool must be a subset of corresponding network ip-pool"